docs/GCS-batchsink.md

```diff
@@ -26,6 +26,12 @@ Properties
 ----------
 **Reference Name:** Name used to uniquely identify this sink for lineage, annotating metadata, etc.
 
+**Use Connection** Whether to use a connection, if a connection is used,
+the credentials does not need to be provided.
+
+**Connection** Name of the connection to use, should use the macro function ${conn:(connection-name)} to provide.
```

Comment:
@itsankit-google You no longer write a macro for the name. You Browse connection and select one. The connection name displays in the Connection field. The field isn't editable.  Maybe just say "Name of the connection to use. Project and service account information will be provided by the connection." (The last sentence is in the GCS source docs)